etc/helm/pachyderm/values.yaml

```diff
@@ -417,6 +417,7 @@ pachd:
     # object storage uploads per Pachd instance.  It is analogous to
     # the --upload-concurrency-limit argument to pachctl deploy.
     uploadConcurrencyLimit: 100
+  pfsGCInterval: 1m
```

Comment:
We should rename this to `storageGCPolling`. We should be consistent with the environment variable names in the Helm chart.

Reply:
We need to gradually make the environment variables be more globally descriptive through namespacing with prefixes (that is what we do with the storage layer configuration by prefixing them with `STORAGE`). The configuration at higher levels should be consistent with the final configuration (the environment variable) because it is going to lead to confusion if we have to track mappings from higher level configuration to lower level configuration.